client/my-sites/plugins/plugin.jsx

```diff
@@ -113,9 +116,8 @@ export default React.createClass( {
 
 	refreshSitesAndPlugins( nextProps ) {
 		this.setState( this.getSitesPlugin( nextProps ) );
-
 		// setTimeout to avoid React dispatch conflicts.
-		setTimeout( () => {
+		this.pluginRefreshTimeout = setTimeout( () => {
```

Comment:
The reason why we can't set the timeout in the controller is because we load the data about the plugin in the plugin component. So we are not passing the controller any more after we get the data. We end up with the slug instead of the name in the title.